SQL/Create-Table_dbo.amar_fishing.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE dbo.amar_fishing
+(
+    [UID] INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    ActionName VARCHAR(150) NOT NULL,
+    ActionNameDrop1 VARCHAR(150)
+        ActionNameDrop1QuantityMin INT,
```

Comment:
Can you fix the spacing on this file too please so they are inline

Reply:
Corrected the spacing on here as well.